homu/html/index.html

```diff
@@ -37,24 +40,39 @@
         </p>
 
         <ul>
-            <li><code>r+ SHA</code>: Accept a PR where the last commit has the given SHA prefix. If you omit the SHA, the last commit at the time of writing is chosen</li>
-            <li><code>r=NAME SHA</code>: Accept a PR on the behalf of NAME</li>
-            <li><code>r-</code>: Unacccept a PR</li>
-            <li><code>p=NUMBER</code>: Set the priority of the accepted PR (defaults to 0)</li>
-            <li><code>rollup</code>: Mark the PR as likely to merge without issue, implies <code>p=-1</code></li>
-            <li><code>rollup-</code>: Unmark the PR as <code>rollup</code></li>
-            <li><code>retry</code>: Signal that the PR is not bad, and should be retried by buildbot</li>
-            <li><code>try</code>: Request that the PR be tested by buildbot, without accepting it</li>
-            <li><code>force</code>: Stop all the builds on the configured builders, and proceed to the next PR</li>
-            <li><code>clean</code>: Clean up the previous build results</li>
+            <li><code>r+ (SHA)</code>: Accept a PR. Optionally, the SHA of the last commit in the PR can be provided as a guard against synchronization issues or mallicious users. Regardless of the form used, PRs will automatically be unaccepted if the contents are changed.</li>
+            <li><code>r=NAME (SHA)</code>: Accept a PR on the behalf of NAME.</li>
+            <li><code>r-</code>: Unacccept a PR.</li>
+            <li><code>p=NUMBER</code>: Set the priority of the accepted PR (defaults to 0).</li>
+            <li><code>rollup</code>: Mark the PR as likely to merge without issue, implies <code>p=-1.</code></li>
+            <li><code>rollup-</code>: Unmark the PR as <code>rollup</code>.</li>
+            <li><code>retry</code>: Signal that the PR is not bad, and should be retried by buildbot.</li>
+            <li><code>try</code>: Request that the PR be tested by buildbot, without accepting it.</li>
+            <li><code>force</code>: Stop all the builds on the configured builders, and proceed to the next PR.</li>
+            <li><code>clean</code>: Clean up the previous build results.</li>
         </ul>
 
-        <h3>Examples</h3>
+        <h4>Examples</h4>
```

Comment:
It was primarily done to cope with IE... Its default font size for `<h1>` was too big in my opinion. After forcing `<h1>`'s the font size, `<h2>`'s became problematic in IE as it looked bigger than `<h1>`! And so on.

As we are not sure of other browsers, I guess it would be safer to assume those styling may be wrong. But, as a respect to the browser defaults, my way to deal with the problem may be severely wrong. In that case, we should get rid of the `<h1>`-`<h3>` styling entirely, as `<h4>`'s size will be bigger than them without an explicit styling. So it is basically an all-or-nothing issue. What's your opinion on this?